### ISSUE TITLE
Personal Performance Rationale

### ISSUE THREAD

**Issue by alexhealy** [alexhealy/portfolio#3]
| | 0 points | 1 points | 2 points | 3 points |
| --- | --- | --- | --- | --- |
| **Patterns** | Barely started | Missing lots of patterns | Has most of the patterns, some missing | All patterns exist and are well done |
| **Personality** | Bland, boring, just another graphic designer | Fairly generic but may be better with visual design | Quality layouts and unique, personable content | Unique and recognizable as you, with engaging, personable content |
| **Text content** | Fake content only | Some real content, some fake | All real content but with lots of errors | Real content, well written, no grammar or spelling errors |
| **Responsiveness** | Not responsive | Works on some screens | Responsive but with lots of awkwardness | Looks great on all screen sizes |
| **Documentation** | No documentation | Very little documentation | Mostly well documented | Amazing! Anybody could understand |
| **Use of systems** | Everything is repeated | Some good use of Jekyll & Patternbot | Lots of good systemization & reuse | Lego! |
| **Code quality** | Barely started | Indentation is barely existent, lots of validation errors, very poor semantics | Decent indentation, just a couple validation errors, decent semantics | Well indented, fully valid, good semantics |

Overall, I believe that I should receive an A on my pattern library.

I was able to design and create a responsive and creative pattern library that showcases my brand and portfolio identity through its colours and designs. I feel that I have successfully used the knowledge learned throughout the past web development courses in order to create this final portfolio piece, meeting all elements needed, such as the use of HTML, CSS & Java, as well as Jekyll, Markdown and YAML. 

I believe that my pattern library can be utilized in my future portfolio, with its clean and organized code that has the ability to be changed and altered with its easy accessibility. 

An area of my pattern-library that could be improved includes a few config descriptions as well as the choice of background colour on my branding page that go along with my logo. It is a little too dark in order for my logo to be shown clearly.    

I did not run into too many major problems however one major problem was implementing my videos as a banner. This included converting the mov. format into the proper mp4. format, however the video still seemed to be having issues. I then changed my code in order for it to connect to my Vimeo video, this seemed to load the videos faster than the mp4 itself. The video code would not respond to the controls, stopping rather than looping. This was solved by re exporting the videos from After Effects into Media Encoder which seemed to fix the issues automatically playing the video when a page was opened. 

In conclusion, I believe I have implemented and used all code necessary in order to complete a responsive pattern library that properly brands my identity. Each section has to capability to be changed and altered easily for future reference and usage.

@thomasjbradley
